content/en/containers/cluster_agent/troubleshooting.md

```diff
@@ -287,6 +290,214 @@ And indeed `824 * 3 replicas = 2472`.
 
 *Disclaimer*: The Datadog Cluster Agent processes the metrics set in different HPA manifests and queries Datadog to get values every 30 seconds, by default. Kubernetes queries the Datadog Cluster Agent every 30 seconds, by default. As this process is done asynchronously, you should not expect to see the above rule verified at all times, especially if the metric varies, but both frequencies are configurable in order to mitigate any issues.
 
+## Cluster Checks
+
+### Kubernetes: find the leader Cluster Agent
+
+When leader election is enabled, only the leader serves Cluster Check configurations to the node-based Agents. If only one replica of the Cluster Agent Pod is running, it is the leader. Otherwise, you can identify the name of the leader in the `datadog-leader-election` `ConfigMap`:
+
+```yaml
+# kubectl get cm datadog-leader-election -o yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+    control-plane.alpha.kubernetes.io/leader: '{"holderIdentity":"cluster-agent-rhttz", ...''
+```
```

Comment:
I think this code example is going wrong

Reply:
It looks fine in the staging though so 🤷‍♀️ https://docs-staging.datadoghq.com/cswatt/cluster-agent-okr/containers/cluster_agent/troubleshooting/#autodiscovery-in-the-cluster-agent

Reply:
hmm i think it's rendering fine, but i'll clean up the end of that example string